tests/utils/test_dist.py

```diff
@@ -1,16 +1,29 @@
 # Copyright 2022 MosaicML Composer authors
 # SPDX-License-Identifier: Apache-2.0
 
+from unittest.mock import patch
+
```

Comment:
very minor nit: can we get rid of this new line and put in same import block?

Reply:
Actually no. The auto formatter is doing this.

Reply:
😭 ok will merge then